### ISSUE TITLE
Issue with keys in code and documentation

### ISSUE THREAD

**Issue by jitujazz** [JohannesKlauss/react-hotkeys-hook#860]
Hi,

I have written a very simple code line in my application

`useHotkeys('ctrl+m', () => alert('CTRL M Clicked!'))`

When I press the `ctrl+m` for first time, it works great. But then, on every click of just the `ctrl` key the alert pops up. I was checking the documentation for a possible solution and even over there too, after clicking on the `a` in the first example, clicking on `shift`, `esc`, `home`, `end`, etc keys, the same alert pops up [Doc Link](https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/basic-usage). I also logged the keyevent in the same example

<img width="846" alt="image" src="https://user-images.githubusercontent.com/9718425/205472248-b9a932a3-ef50-4fd1-aa9d-afa416313aac.png">

Am I missing anything here? Am I doing anything wrong here?

**Comment by nirtamir2** [comment]
Same here.
I end up add temporary hack to check the keys manually:

```ts
const globalHotkeysRef = useHotkeys<HTMLDivElement>(
  ["meta+f"],
  (keyboardEvent, hotkeysEvent) => {
    /**
     * I think there is a bug in the library because after the first meta+f trigger,
     * when I press meta in the shortcut scope, it triggers even without "f" key press
     * This HACK is to check my keyboard event manually
     *
     * Also: notice meta does not work well on windows:
     * https://github.com/JohannesKlauss/react-hotkeys-hook/issues/849
     *
     * Cleaning also seems to have problems:
     * https://github.com/JohannesKlauss/react-hotkeys-hook/issues/857
     *
     */
    if (
      hotkeysEvent.meta != null &&
      hotkeysEvent.meta &&
      keyboardEvent.key === "f"
    ) {
      setIsOpen(true);
    }
  },
  // This prevents the browser's default search toolbar
  { preventDefault: true },
  [setIsOpen]
);
```

**Comment by karaleina** [comment]
I have the same issue.

**Comment by JohannesKlauss** [comment]
Yes, this is a known issue with macOS. I am trying to come up with a workaround for that weird behavior

**Comment by JohannesKlauss** [comment]
Fixed in `4.1.0.`
